JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.owl

```diff
@@ -84,12 +97,74 @@
     </owl:Class>
+    
+
+
+    <!-- https://www.theworldavatar.com/kg/ontotimeseries/CumulativeTimeSeries -->
+
+
+    <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontotimeseries/CumulativeTimeSeries">
+        <rdfs:subClassOf rdf:resource="https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Time series of cumulative data</rdfs:comment>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontotimeseries/</rdfs:isDefinedBy>
+    </owl:Class>
+    
+
+
+    <!-- https://www.theworldavatar.com/kg/ontotimeseries/CumulativeTotalTimeSeries -->
+
+
+    <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontotimeseries/CumulativeTotalTimeSeries">
+        <rdfs:subClassOf rdf:resource="https://www.theworldavatar.com/kg/ontotimeseries/CumulativeTimeSeries"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Time series of running sum</rdfs:comment>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontotimeseries/</rdfs:isDefinedBy>
+    </owl:Class>
+    
+
+
+    <!-- https://www.theworldavatar.com/kg/ontotimeseries/InstantaneousTimeSeries -->
+
+
+    <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontotimeseries/InstantaneousTimeSeries">
+        <rdfs:subClassOf rdf:resource="https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Time series where the data is the measurement value at each time step</rdfs:comment>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontotimeseries/</rdfs:isDefinedBy>
+    </owl:Class>
+    
+
+
+    <!-- https://www.theworldavatar.com/kg/ontotimeseries/StepwiseCumulativeTimeSeries -->
+
+
+    <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontotimeseries/StepwiseCumulativeTimeSeries">
+        <rdfs:subClassOf rdf:resource="https://www.theworldavatar.com/kg/ontotimeseries/CumulativeTimeSeries"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Time series where data is cumulative between time steps</rdfs:comment>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontotimeseries/</rdfs:isDefinedBy>
+    </owl:Class>
+    
+
+
+    <!-- https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries -->
+
+
+    <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries">
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontotimeseries/</rdfs:isDefinedBy>
+    </owl:Class>
+    
+
+
+    <!-- http://www.w3.org/2006/time#TemporalDuration -->
+
```

Comment:
Looking at `https://www.w3.org/TR/owl-time/#time:TemporalDuration` in more detail, I think this not the right class (at least it makes it unnecessary complex). 
I would suggest using `https://www.w3.org/TR/owl-time/#time:Duration` (which is a subclass of TemporalDuration), which can be directly related to the actual duration value and unit using `https://www.w3.org/TR/owl-time/#time:numericDuration` and `https://www.w3.org/TR/owl-time/#time:unitType`, respectively.
I would suggest to include all those triples into our ontology (setting the defined by column to `https://www.w3.org/TR/owl-time/`) so that people directly see how to use it. Maybe we could even include the temporal units just for reference